### PR TITLE
feature(perf-i4i): New perf throughput non-shard aware i4i tests

### PR DESCRIPTION
**Pull request by aleksbykov**
Add 2 new jenkins files and config yaml files for master and enterprise performance i4i testing

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: jenkins-pipelines/perf-regression-throughput-non-shard-aware-30min-i4i-enterprise.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    region: "us-east-1",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: '''["test-cases/performance/perf-regression.100threads.30M-keys-i4i-enterprise.yaml","configurations/perf-loaders-non-shard-aware-config.yaml", "configurations/run_db_node_benchmarks.yaml", "configurations/disable_speculative_retry.yaml"]''',
+    sub_tests: ["test_write", "test_read", "test_mixed"],
+    test_email_title: "throughput - non-shard-aware",
```
Comment (fgelcer) [inline jenkins-pipelines/perf-regression-throughput-non-shard-aware-30min-i4i-enterprise.jenkinsfile:12]: IIUC it doesn't work with the new changes by @temichus on #6221

File: jenkins-pipelines/perf-regression-throughput-non-shard-aware-30min-i4i.jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    region: "us-east-1",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: '''["test-cases/performance/perf-regression.100threads.30M-keys-i4i.yaml","configurations/perf-loaders-non-shard-aware-config.yaml", "configurations/run_db_node_benchmarks.yaml", "configurations/disable_speculative_retry.yaml"]''',
+    sub_tests: ["test_write", "test_read", "test_mixed"],
+    test_email_title: "throughput - non-shard-aware",
```
Comment (fgelcer) [inline jenkins-pipelines/perf-regression-throughput-non-shard-aware-30min-i4i.jenkinsfile:12]: same here

**Review by fgelcer — DISMISSED** [review]
little concern about recent changes being proposed, but other than that, LGTM

File: jenkins-pipelines/perf-regression-throughput-non-shard-aware-30min-i4i-enterprise.jenkinsfile (hunk 1 above)
Comment (aleksbykov) [reply jenkins-pipelines/perf-regression-throughput-non-shard-aware-30min-i4i-enterprise.jenkinsfile:12]: I checked , it should n't affect on throughput test email subject

**Review by temichus — CHANGES_REQUESTED** [review]

**Review by temichus — DISMISSED** [review]
other LGTM

File: test-cases/performance/perf-regression.100threads.30M-keys-i4i.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+test_duration: 300
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=50 -pop seq=1..30000000"
+prepare_write_cmd: "cassandra-stress write no-warmup cl=QUORUM n=30000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..30000000"
+stress_cmd_r: "cassandra-stress read no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=60 -pop 'dist=gauss(1..30000000,15000000,1500000)' "
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=50 -pop 'dist=gauss(1..30000000,15000000,1500000)' "
+stress_multiplier: 6
```
Comment (roydahan) [inline test-cases/performance/perf-regression.100threads.30M-keys-i4i.yaml:7]: As I disucssed with @aleksbykov, we will split this into:
stress_read_multiplier
stress_write_multiplier
stress_mixed_multiplier

And we will support different num of processes per command.

**Comment by roydahan** [comment]
@aleksbykov you need to remove the commit of split_stress_multiplier.

File: test-cases/performance/perf-regression.100threads.30M-keys-i4i-enterprise.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+test_duration: 300
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..30000000"
+prepare_write_cmd: "cassandra-stress write no-warmup cl=QUORUM n=30000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..30000000"
```
[the remark this answers is not in the document]
Comment (roydahan) [reply test-cases/performance/perf-regression.100threads.30M-keys-i4i-enterprise.yaml:4]: @aleksbykov did we review the results for enterprise?
I see the changes you did only for the OSS one.

**Comment by aleksbykov** [comment]
Fix according comments. @roydahan @temichus @fgelcer could you take a look

**Review by roydahan — APPROVED** [review]

**Comment by roydahan** [comment]
Backported.

**Pull request merged**